session.py

```diff
@@ -0,0 +1,671 @@
+        # search for the match sets
+        while  next_setURL != None:
+            results = json.loads(response_get_set.text).get('results')
+            for result in results:
+                if result.get('exerciseday') == trainingId:
+                    match_sets.append(result)
+            next_setURL = json.loads(response_get_set.text).get('next')
+            if next_setURL != None:
+                response_get_set = self.s.get(next_setURL, headers= self.headers)
+
+        return match_sets
+    
+    def get_meals(self, planId):
+        """ Get the meals of a specific nutritionplan ( plan )
+
+            args:
+                planId ( int ): plan id of the meals you want to get
+            
+            returns:
+                list: the meals of the workout
+        """
+        # create the URL
+        get_mealURL = self.base_URL + "meal/"
+
+        # make the request
+        response_get_meal = self.s.get(get_mealURL, headers= self.headers)
+
+        match_meals =  []
+        next_mealURL = 0
+        
+        # search for the match meals
+        while  next_mealURL != None:
+            results = json.loads(response_get_meal.text).get('results')
+            for result in results:
+                if result.get('plan') == planId:
+                    match_meals.append(result)
+            next_mealURL = json.loads(response_get_meal.text).get('next')
+            if next_mealURL != None:
+                response_get_meal = self.s.get(next_mealURL, headers= self.headers)
+
+        return match_meals
+    
+    def get_mealitems(self, mealId):
+        """ Get the mealitems of a specific meal
+
+            args:
+                mealId ( int ): meal id of the mealitems you want to get
+            
+            returns:
+                list: the mealitems of the meal
+        """
+        # create the URL
+        get_mealitemURL = self.base_URL + "mealitem/"
+
+        # make the request
+        response_get_mealitem = self.s.get(get_mealitemURL, headers= self.headers)
+
+        match_mealitems =  []
+        next_mealitemURL = 0
+
+        # search for the match mealitems 
+        while  next_mealitemURL != None:
+            results = json.loads(response_get_mealitem.text).get('results')
+            for result in results:
+                if result.get('meal') == mealId:
+                    match_mealitems.append(result)
+            next_mealitemURL = json.loads(response_get_mealitem.text).get('next')
+            if next_mealitemURL != None:
+                response_get_mealitem = self.s.get(next_mealitemURL, headers= self.headers)
+
+        return match_mealitems
+
+    
+    def check(response):
+        """ Check a request
+
+            args:
+                response: the response of a request
+            
+            returns:
+                str: whether the request was successful or not
+        """
+        if (response.status_code == 200):
+            return "Successful get request !"
```

Comment:
Here you should use some kind of booleans in order to easier check the return of this function. At the moment you have to parse the strings in orders to see if the request was done successfully.